packages/lesswrong/server/conversationUnreadMessages.ts

```diff
@@ -1,22 +1,25 @@
 import { createPostgresView } from "./postgresView";
 
-createPostgresView(`
-  CREATE OR REPLACE VIEW "ConversationUnreadMessages" AS
-  SELECT q."conversationId", q."userId", EXISTS (
-    SELECT *
-    FROM "Notifications" n
-    INNER JOIN "Messages" m ON
-      n."userId" = q."userId"
-      AND n."documentId" = m."_id"
-      AND n."documentType" = 'message'
-      AND n."emailed" IS NOT TRUE
-      AND n."waitingForBatch" IS NOT TRUE
-      AND n."deleted" IS NOT TRUE
-      AND n."viewed" IS NOT TRUE
-      AND m."conversationId" = q."conversationId"
-    ) "hasUnreadMessages"
-  FROM (
-    SELECT "_id" "conversationId", UNNEST("participantIds") "userId"
-    FROM "Conversations"
-  ) q
-`);
+createPostgresView(
```

Comment:
This query is unchanged - we're just adding the new extra parameters for `createPostgresView`